data/master.tsv

```diff
@@ -96,6 +96,7 @@ Animal:Ursidae.N	Animal:bear.N	A large beast of prey of the family Ursidae, rela
 Animal:Vespinae.N	Animal:wasp.N	A flying stinging insect related to the bee, which is usually coloured yellow and black.	1517	3-823		
 Animal:Vulpes.N	Animal:fox.N	A carnivorous relatively small canine of the species Vulpes Vulpes.	1312	3-74	Fuchs::N	964
 Animal:wool.N	Clothing/Grooming:wool.N	The soft, curly hair that forms the fleece of sheep, lamas and other animals and which is used to produce clothing.	964	6-22	Wolle::N	1403
+Animal:zebra.N	Animal:zebra.N	Any of several species of genus Equus with distinctive black-and-white striped coats, native to Africa.	3675			

```

Comment:
You are right that zebra is not one species, but there is the subgenus Hippotigris for all zebras. So the id should be `Animal:Equus (Hippotigris).N`. That seems to be the standard way to write it.